passlib/apps.py

```diff
@@ -138,6 +138,14 @@ def _load_master_config():
 _django21_schemes.remove("django_bcrypt")
 django21_context = LazyCryptContext(schemes=_django21_schemes)
 
+# -----------------------------------------------------------------------
+# 3.1
+# -----------------------------------------------------------------------
```

Comment:
Please let's not use this style of comments, IMO they're pretty bulky and hurt readability:
```suggestion
# Django >=3.1
```

Reply:
100% agree